ferveo-python/examples/server_api.py

```diff
@@ -48,14 +49,15 @@
 transcripts_ser = [bytes(transcript) for _, transcript in messages]
 transcripts_deser = [Transcript.from_bytes(t) for t in transcripts_ser]
 
-agg_transcript_ser = bytes(pvss_aggregated)
-agg_transcript_deser = AggregatedTranscript.from_bytes(agg_transcript_ser)
-
 # In the meantime, the client creates a ciphertext and decryption request
 msg = "abc".encode()
 aad = "my-aad".encode()
 ciphertext = encrypt(msg, aad, dkg.final_key)
 
+# The client can serialize/deserialize ciphertext for transport
+ciphertext_ser = bytes(ciphertext)
+ciphertext_deser = Ciphertext.from_bytes(ciphertext_ser)
```

Comment:
Do you want to use `ciphertext_deser` on line 72 instead of `ciphertext`? Similar to what we did for `agg_transcript_deser`.